stable/redis-ha/templates/redis-sentinel-deployment.yaml

```diff
@@ -11,6 +11,9 @@ spec:
         name: {{ template "fullname" . }}-sentinel
 {{ include "labels.standard" . | indent 8 }}
     spec:
+      {{- if .Values.rbac.create }}
+      serviceAccountName: {{ template "fullname" . }}
```

Comment:
`{{ template "redis-ha.serviceAccountName" . }}` and remove the `if` statement around it.